glotaran/builtin/models/kinetic_spectrum/test/test_spectral_relations.py

```diff
@@ -113,16 +113,20 @@ def test_spectral_relation():
     scheme = Scheme(model=model, parameter=parameter, data=dataset, nfev=20)
     result = optimize(scheme)
 
+    for label, param in result.optimized_parameter.all():
+        if param.vary:
+            assert np.allclose(param.value, parameter.get(label).value, rtol=1e-1)
```

Comment:
Will make a note to revisit proper relative tolerance to be used here, see #495.